Removing ondrej ppa/adding refs to server hardening (#3858)
owncloud/docs: modules/admin_manual/pages/installation/manual_installation/manual_installation.adoc

This document describes:

* How to prepare your server
* Prerequisites and how to download ownCloud
* Installation of ownCloud

The following descriptions focus on the Ubuntu distribution. Even if we try to make these steps
as easy as possible by offering ready to use commands and scripts, you need to have sufficient

> knownledge about administrating a server environment which provides web services.

Reviewer: ```suggestion
knowledge about administrating a server environment which provides web services.
```

> IMPORTANT: This document does not offer proposals how to secure your server. Therefore, we strongly recommend to check out the xref:configuration/server/harden_server.adoc[Hardening and Security Guidance] before the installation and to keep it at hand through out.

Reviewer: ```suggestion
IMPORTANT: This document does not offer proposals about how to secure your server. Therefore, we strongly recommend checking out the xref:configuration/server/harden_server.adoc[Hardening and Security Guidance] before the installation and to keep it on hand through out.
```

=== Install a Database

> If you do not already have installed a supported database, follow the

Reviewer: ```suggestion
If you do not already have a supported database installed, follow the
```

> xref:installation/manual_installation/manual_installation_db.adoc[Manual Installation Databases guide].

Reviewer: ```suggestion
xref:installation/manual_installation/manual_installation_db.adoc[Manual Database Installation guide].
```

=== Configure the Web Server

> To configure your Apache web server for the use with ownCloud, follow the

Reviewer: ```suggestion
To configure your Apache web server for use with ownCloud, follow the
```

xref:installation/manual_installation/manual_installation_apache.adoc[Apache preparation guide].

== Installation of ownCloud Binaries

To install ownCloud binaries, you have to download the required package. After doing so,
you can perform the following steps manually or use the provided scripts. These scripts are convenient
since they can also be used for upgrading which eases the process a lot. 
 
=== Download ownCloud

Before downloading ownCloud, change to a directory where you want to save

> the file temporarily. This can be, for example `/tmp`. In further examples, we use tar archives or the complete ownCloud. The name for the complete archive looks like this:

Reviewer 2: ```suggestion
the file temporarily. This can be, for example `/tmp`. In further examples, we use tar archives or the complete ownCloud package. The name for the complete archive looks like this:
```
